src/main/java/minicraft/item/FurnitureItem.java

```diff
@@ -25,7 +25,6 @@ protected static ArrayList<Item> getAllInstances() {
 		items.add(new FurnitureItem(new Spawner(new Skeleton(1))));
 		items.add(new FurnitureItem(new Spawner(new Snake(1))));
 		items.add(new FurnitureItem(new Spawner(new Knight(1))));
-		items.add(new FurnitureItem(new Spawner(new AirWizard())));
 
```

Comment:
The AirWizard spawner was removed. Does that mean it is no longer possible to spawn it in creative?

Reply:
Bugs start to happen when there are more than 1 boss, therefore I removed the Spawner to solve this issue. As for creative, that is what the totem of air is for.